lalrpop/src/lr1/error/test.rs

```diff
@@ -180,8 +180,69 @@ VarDecl = "let";
     let err = build_states(&grammar, nt("Func")).unwrap_err();
     let mut cx = ErrorReportingCx::new(&grammar, &err.states, &err.conflicts);
     let conflicts = super::token_conflicts(&err.conflicts);
-    for conflict in &conflicts {
+    for conflict in conflicts.iter().flatten() {
         println!("conflict={:?}", conflict);
         cx.classify(conflict);
     }
 }
+
+fn verify_errors(
+    grammar: Grammar,
+    pub_state: &str,
+    unique_conflicts: usize,
+    terminal_count: usize,
+    text: &str,
+) {
+    use crate::message::{Content, Message};
+    use ascii_canvas::AsciiCanvas;
+    let _tls = Tls::test();
+    let _lr1_tls = Lr1Tls::install(grammar.terminals.clone());
+    let err = build_states(&grammar, nt(pub_state)).unwrap_err();
+    let mut cx = ErrorReportingCx::new(&grammar, &err.states, &err.conflicts);
+    let conflicts = super::token_conflicts(&err.conflicts);
+    assert_eq!(conflicts.len(), unique_conflicts); // One group of conflicts
+    assert_eq!(conflicts[0].len(), terminal_count); // terminal count
```

Comment:
Nit: Unsure how much more extensively you plan to use this in later pr's but converting this to check terminal count with the sum of all conflicts is probably more robust.